sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -178,7 +179,12 @@ private[hive] class SparkExecuteStatementOperation(
           }
           curCol += 1
         }
-        resultRowSet.addRow(row.toArray.asInstanceOf[Array[Object]])
+        val resultRow = row.map {
```

Comment:
I have to convert Java 8 classes to `java.sql.Timestamp`/`Date` for correct formatting. I cannot use `TimestampFormatter` here because the conversions to strings can be performed in `ColumnBuffer` in `hive-serde`.

Reply:
can you add a code comment? Say that the results are passed to Hive and Hive only accepts  `Timestamp`/`Date`